src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {   //ok to leave as public?
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+    public String getTypeIcon() {
+        return "NULL";
+    }
+    public String taskTypeIcon() { return "[" + getTypeIcon() + "]";}
```

Comment:
Even though these are simple 1 liners, please do still follow the proper style (aka Egyptian Style).